### DISCUSSION TITLE
AFN Guide

### DISCUSSION THREAD

**Discussion by baliebox4916** [iptv-org/community#827] [Q&A]
### Channel ID

AFNGuide.us

### Notes

One Last Addition to the AFN Lineup and That is the AFN Guide

I May Add More as Time Goes on

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19819
